app/platform/fabric/config.json

```diff
@@ -2,7 +2,10 @@
 	"network-configs": {
 		"test-network": {
 			"name": "Test Network",
-			"profile": "./connection-profile/test-network.json"
+			"profile": "./connection-profile/test-network.json",
+			"blockCount": 5,
+			"purgeMode":"NONE",
```

Comment:
I assume we will now have the following purgemode NONE, TIME,BLOCKCOUNT, are we also handling the NONE case as well (which means no purge is needed).

Reply:
yes, we are handling the NONE case(* No purge) too.